curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/6449842c6f6c84261075e4c9.md

```diff
@@ -55,7 +63,7 @@ Your callback function should have a `cell` parameter.
 assert.match(code, /const\s+evalFormula\s*=\s*\(\s*x\s*,\s*cells\s*\)\s*=>\s*{\s*const\s+idToText\s*=\s*(\(\s*id\s*\)|id)\s*=>\s*cells\.find\(\s*(\(\s*cell\s*\)|cell)\s*=>/);
 ```
 
-Your callback function should use an implicit return.
+Your `.find()` callback should use an implicit return.
```

Comment:
I made the suggestion because it's made in other fCC challenges, like the [Shopping Cart](https://github.com/freeCodeCamp/freeCodeCamp/blob/c98e1803a375aa08ada3189e2b41301b4446ac08/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-oop-by-building-a-shopping-cart/63f02a4ef92d711ec1ff618c.md?plain=1#L45) and the [Spreadsheet challenge itself](https://github.com/freeCodeCamp/freeCodeCamp/blob/c98e1803a375aa08ada3189e2b41301b4446ac08/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/64345b810a6e481e5e326849.md?plain=1#L40).

Also, probably being more explicit it's more helpful than not.

Reply:
Ok, @huyenltnguyen and @juancaorg 
If it's ok with y'all I would proceed as it follows:

- For this specific PR I would revert this last change
- If we want to clarify better the hints we could open a new issue and discuss on that and eventually make the changes needed.

What do you think?

Reply:
Sure! It is fine by me.